content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,289 @@
+context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+Now we just need to run the python script which includes this code to cache and
+register the model.
+
+If you are following our example repository then the training script is included
+in a dvc pipeline [we prepared](/doc/start/experiments/experiment-pipelines) in
+the Experiment Management guide, so there we just call `dvc exp run` to run the
+entire experiment pipeline.
+
+We now commit the result to git (and push it to our git remote) and the new
+model will show up in the model registry in Studio. We will also push the
+resulting artifact from DVC cache to our DVC Remote by calling `dvc push`.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provided in the call and edits the `dvc.yaml` file which will now contain the
+following lines
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your git remote it is parsed by Studio and
+it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+To get acquainted with how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## DVC Model registry overview
+
+In this guide, we will be using DVC Studio to manage our model registry. DVC
+Studio is a web application which integrates all Iterative tools like DVC and
+adds extra functionality aimed at sharing and collaboration. You can find out
+more about it [here](/doc/studio).
```

Comment:
```suggestion
In this guide, we will be using [DVC Studio](https://studio.iterative.ai)
to manage our model registry. Studio enables you to see models across all 
projects, manage their lifecycle, and download them with only a token. 
You can find out more about it [here](/doc/studio).
```

I think the intro should be about why we use Studio here.

Reply:
Makes sense